package.json

```diff
@@ -16,14 +16,19 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@google-cloud/language": "^0.11.0",
```

Comment:
@meghanprestemon please remove the `^` to lock this dependency in place, thanks!